haystack/telemetry.py

```diff
@@ -291,6 +291,7 @@ class NonPrivateParameters:
         "type",
         "uptime",
         "run_total",
+        "run_total_window",
```

Comment:
@julian-risch this is the new `run_total_window` attribute being added to allowed list of attribtues